doc/ref/chap-type-method.md

```diff
@@ -138,27 +138,77 @@ rather than its value.
 
 ### append()
 
+Add an element to a list.
+
+    var fruits = :|apple banana pear|
+    call fruits -> append("orange") # returns null
```

Comment:
I think we want to encourage a style of no space for ->`, but spaces for `=>` to make it look more different?

```
call fruits->append('orange')
```

vs.

```
var v = d => values() => reversed()
```

Because `=>` can be chained and `->` can't.  `->` generally has no return value

What do you think?